cmake/helpers.cmake

```diff
@@ -232,9 +232,8 @@ function(add_umf_target_compile_options name)
             PRIVATE -fPIC
                     -Wall
                     -Wextra
-                    -Wpedantic
```

Comment:
I you want to add this change in this PR, it should be in a sep. commit

Reply:
Done